src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -312,6 +318,10 @@ def nlcd_soils(sjs_result):
         g2 = settings.SOIL_GROUP[g]
         ng_count[(n, g2)] = count + ng_count.get((n, g2), 0)
 
+    # Immediately return an error for AOIs without enough land cover
+    if len(nt_count.values()) == 0:
```

Comment:
Switched this to check `ngt_count.values()` and return immediately if the length of that's 0.